### PR TITLE
Configure admin template only if AdminBundle is present

### PR DESCRIPTION
**Pull request by Wharenn**
I am targeting this branch, because this is a non-BC fix

## Changelog

```markdown
### Fixed
- Fixed The service definition "sonata.page.admin.page" does not exist.

```

## Subject

Custom templates for admin are trying to access the ``sonata.page.admin.page`` service in all cases, but this service is only in the container if AdminBundle is loaded in the app Kernel. 

Inn this PR, admin templates configuration is done only if the AdminBundle is present.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "3.x-dev"
+            "dev-master": "4.x-dev"
```
Comment (Soullivaneuh) [inline composer.json]: This must not be changed.

**Comment by Soullivaneuh** [comment]
> I am targeting this branch, because this is a non-BC fix

Yeah but this branch is the branch for BC fixes and features. :-)

BTW, could you make it BC?

File: DependencyInjection/SonataPageExtension.php [hunk 2]
```diff
@@ -54,6 +54,8 @@ public function load(array $configs, ContainerBuilder $container)
             if (!$config['direct_publication']) {
                 $container->removeDefinition('sonata.page.admin.extension.snapshot');
             }
+
+            $this->configureTemplatesAdmin($container, $config);
```
Comment (Soullivaneuh) [inline DependencyInjection/SonataPageExtension.php]: I'm not really sure this is BC break.

If the admin-bundle is not loaded, who the f*ck will use admin templates? :-)

cc @sonata-project/contributors

Comment (greg0ire) [inline DependencyInjection/SonataPageExtension.php]: People doing that deserve to get the BC break in a new minor.

**Comment by Wharenn** [comment]
There was a branch mismatch in my side, closed for #681.
Thank you

**Comment by Soullivaneuh** [comment]
Don't close, this is the right branch!

**Comment by Soullivaneuh** [comment]
Just remove not wanted commit and we will be good. :+1:

File: UPGRADE-4.0.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+UPGRADE FROM 3.x to 4.0
```
Comment (Soullivaneuh) [inline UPGRADE-4.0.md]: Please remove this file.

**Comment by Soullivaneuh** [comment]
#681

**Pull request closed without merging**